### PR TITLE
Demo Proposal: Proposal to do great things for Algorand ecosystem

### PR DESCRIPTION
**Pull request by algohodl**
Thank you for considering my proposal

### CODE REVIEW

File: Proposals/xgov-10.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+period: 7
+title: Proposal to do great things for Algorand ecosystem
+author: algohodl (algohodl)
```
Comment (SudoWeezy) [inline Proposals/xgov-10.md:4]:
```suggestion
author: algohodl (@algohodl)
```

**Review by SudoWeezy — COMMENTED** [review]
Thanks for your proposal @algohodl. I have suggested some formatting changes to have a proper structure.

**Comment by No-Cash-7970** [comment]
Now I know why this proposal is here after watching the video guide.

**Comment by nonfungibleab** [comment]
@algohodl could you add - Demo Proposal to your proposal title, please?

**Pull request closed without merging**